packages/valory/services/trader_omen_gnosis/service.yaml

```diff
@@ -79,7 +79,6 @@ models:
       redeem_round_timeout: 3600.0
       contract_timeout: 300.0
       file_hash_to_strategies_json: ${FILE_HASH_TO_STRATEGIES_JSON:list:[["bafybeihufqu2ra7vud4h6g2nwahx7mvdido7ff6prwnib2tdlc4np7dw24",["bet_amount_per_threshold"]],["bafybeif55cu7cf6znyma7kxus4wxa2doarhau2xmndo57iegshxorivwmq",["kelly_criterion"]]]}
-      strategies_kwargs: ${STRATEGIES_KWARGS:list:[["bet_kelly_fraction",1],["floor_balance",500000000000000000],["bet_amount_per_threshold",{"0.0":0,"0.1":0,"0.2":0,"0.3":0,"0.4":0,"0.5":0,"0.6":60000000000000000,"0.7":80000000000000000,"0.8":160000000000000000,"0.9":1000000000000000000,"1.0":1000000000000000000}]]}
```

Comment:
@Adamantios This export is causing issues on Pearl when exporting variables. Perhaps open an issue on Open Autonomy. The approach to make it work is to remove the service override and use directly the agent configuration.

Reply:
Are you sure you can even use override on Pearl? From https://github.com/valory-xyz/trader/pull/221#issuecomment-1945981266 I understand that it is not possible.

Reply:
Pearls seems to be using a custom method for overrides, but it's failing for STRATEGIES_KWARGS.

Reply:
These work fine. The problem with `STRATEGIES_KWARGS` is explained in the description of this PR. https://github.com/valory-xyz/trader/pull/271#issue-2344083412